napari/components/viewer_model.py

```diff
@@ -904,6 +907,14 @@ def _add_layers_with_plugins(
             _data = _unify_data_and_user_kwargs(
                 data, kwargs, layer_type, fallback_name=basename
             )
+
+            if getattr(_data[0], 'ndim', None) == 1 and _data[2] in (
+                'image',
+                'labels',
+            ):
+                LOGGER.warning(f'Skipping 1D image: {filename}')
```

Comment:
Warnings should be also translated, more docs here https://napari.org/guides/stable/translations.html.

Also we don't tend to use `LOGGER` but maybe we should more. @tlambert03 what do you think about this?

Reply:
I changed this to `warnings.warn()` and also added the translation given that a user will see that. I also added `with pytest.warns(UserWarning)` in the tests that needed it.